lib/enrollment.rb

```diff
@@ -1,35 +1,10 @@
-require 'pry'
-require_relative 'district'
 class Enrollment
-  attr_accessor :name, :school_data #:high_school_graduation_rates,
+  attr_accessor :name, :high_school_graduation_rates
+  attr_reader :kindergarten_participation
```

Comment:
Conservatively setting this attribute as a getter alone is a questionable decision, but this addressed a rake sanitation:all warning.